tests/telemetry/test_series.py

```diff
@@ -0,0 +1,48 @@
+import time
+
+from ddtrace.internal.telemetry.data.host import get_hostname
+from ddtrace.internal.telemetry.data.metrics import Series
+
+
+def test_series_to_dict():
+    series = Series("test.metric")
+
+    assert series.to_dict() == {
+        "metric": "test.metric",
+        "points": [],
+        "tags": {},
+        "type": Series.COUNT,
+        "common": False,
+        "interval": None,
+        "host": get_hostname(),
+    }
+
+
+def test_series_add_tag():
+    series = Series("test.metric")
+
+    series.add_tag("foo", "bar")
+    assert series.tags == {"foo": "bar"}
+
+
+def test_series_add_point():
+    series = Series("test.metric")
+
+    series.add_point(111111)
+    series.add_point(222222)
+
+    curr_time_in_secs = int(time.time())
+
+    # two points were added to the series object
+    assert len(series.points) == 2
+    # each point should be tuple with two integers (timestamp, value)
+    assert len(series.points[0]) == 2
+    assert len(series.points[1]) == 2
+
+    # assert each timestamp contains a time in the last 10 seconds
+    assert curr_time_in_secs - series.points[0][0] < 10
+    assert curr_time_in_secs - series.points[1][0] < 10
```

Comment:
yeah, this is probably showcasing the issue above and why you should just mock over `time.time()`, then these can be static assertions for a specific timestamp we want.

e.g. what if the method was modified to do `int(time.time()) - 5`, the timestamp would be wrong, but this test would still pass. (silly example, but example why we should be explicit/clear in test cases)